app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -18,9 +18,8 @@ module.exports = React.createClass
     # console.log 'Old marks were', @state.oldSetOfMarks
     newSetOfMarks = []
     # Automatically select new marks.
-    annotations = nextProps.classification?.annotations ? []
-    annotation = annotations[annotations.length - 1]
-    if annotation?
+    annotation = nextProps.annotation
```

Comment:
This was changed because taking the last `classification.annotations` will give a "combo annotation" (an annotation that has a `value` as an array of `annotations`), but `nextProps.annotation` will give the inner annotation only (i.e. just the drawing annotation).